exchanges/bitmex/bitmex_wrapper.go

```diff
@@ -332,46 +333,50 @@ func (b *Bitmex) FetchOrderbook(p currency.Pair, assetType asset.Item) (*orderbo
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (b *Bitmex) UpdateOrderbook(p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
+	book := &orderbook.Base{
+		ExchangeName: b.Name,
+		Pair:         p,
+		AssetType:    assetType,
+	}
+
 	if assetType == asset.Index {
-		return nil, common.ErrFunctionNotSupported
+		return book, common.ErrFunctionNotSupported
```

Comment:
Given this: https://github.com/thrasher-corp/gocryptotrader/pull/530#discussion_r549572444
Is there anything we can do? book.Verify will ensure that an error will get displayed in some form though. Not super important

Reply:
So going off that comment, this is just checking because tickers are sometimes supported for index currencies. There's not much we can do but this ensures we don't do a HTTP request if not needed. The syncer might need to be updated to adjust for this.